### ISSUE TITLE
Possibilité d'importer un lien d'image afin de changer le fond

### ISSUE THREAD

**Issue by racloody** [autiinpu2/timer-sonneries#29]
Il serait vraiment sympa d'avoir la possibilité dans une section "Paramètres avancés" de pouvoir drop un lien d'image ou même flux vidéo qui serait mis en fond

**Comment by yannis300307** [comment]
Oui. Pas forcement "Paramètres avancés" car on n'aurait pas assez de paramètres. Mais cela est risqué car si le contenu au bout du lien change, on pourrait avoir de mauvaises surprises. Il faudrait enregistrer un hash du contenu pour mettre un avertissement si il est changé.
